packages/api/src/brevo-contact/brevo-contact.resolver.ts

```diff
@@ -0,0 +1,41 @@
+import { Args, Int, Mutation, Query, Resolver } from "@nestjs/graphql";
+
+import { BrevoContactsApiService } from "../brevo/brevo-contact-api.service";
+import { BrevoContactsService } from "./brevo-contacts.service";
+import { BrevoContact } from "./dto/brevo-contact";
+import { BrevoContactUpdateInput } from "./dto/brevo-contact.input";
+import { BrevoContactsArgs } from "./dto/brevo-contacts.args";
+import { PaginatedBrevoContacts } from "./dto/paginated-brevo-contact";
+
+@Resolver(() => BrevoContact)
+export class BrevoContactResolver {
+    constructor(
+        private readonly brevoContactsService: BrevoContactsService,
+        private readonly brevoContactApiService: BrevoContactsApiService, // @Inject(CONFIG) private readonly config: Config,
```

Comment:
// @Inject(CONFIG) private readonly config: Config,
the comment can be removed

Reply:
Removed in [a04b768](https://github.com/vivid-planet/comet-brevo-module/pull/4/commits/a04b768087c44aa5533f33cb84789b64bae9367d)